_episodes/08-maintenance.md

```diff
@@ -17,34 +17,25 @@ This episode describes the processes used to maintain our lessons.
 
 ## Maintainers
 
-Each Software or Data Carpentry lesson has one or two maintainers,
+Each Carpentry lesson has multiple Maintainers,
 who are responsible for making sure issues and change requests are looked at,
 and who have final say over what is included in the lesson.
-Together,
-they also decide on changes to the lesson templates,
-release procedure,
-and other mechanical aspects of lesson production.
 They are *not* responsible for writing lesson content or deciding what lessons ought to exist:
 the former comes from the community,
-and the latter from the Executive Directors and Steering Committees of Software and Data Carpentry.
+and the latter from the Carpentry Executive Council and Curricular Advisory Committees.
```

Comment:
Can you add links to who are the members of each group?

Reply:
@rgaiacs - There aren't yet webpages for these (to my knowledge). The Curricular Advisory Committees have not yet been determined and the Steering Committee pages do not reflect the incoming Executive Council. The joint Executive Council will likely be on the joint Carpentries webpage, which is not yet live.